unison-share-api/src/Unison/Server/Backend.hs

```diff
@@ -348,16 +434,15 @@ lsAtPath codebase mayRootBranch absPath = do
   lsBranch codebase b
 
 findShallowReadmeInBranchAndRender ::
-  Width ->
   Rt.Runtime Symbol ->
   Codebase IO Symbol Ann ->
   PPED.PrettyPrintEnvDecl ->
   V2Branch.Branch m ->
   Backend IO (Maybe Doc.Doc)
-findShallowReadmeInBranchAndRender width runtime codebase ppe namespaceBranch =
+findShallowReadmeInBranchAndRender runtime codebase ppe namespaceBranch =
   let renderReadme :: PPED.PrettyPrintEnvDecl -> Reference -> IO Doc.Doc
       renderReadme ppe docReference = do
-        (_, _, doc) <- renderDoc ppe width runtime codebase docReference
+        doc <- evalDocRef runtime codebase docReference <&> Doc.renderDoc ppe
```

Comment:
Turns out that the render width isn't actually used in the renderer at all; we should probably use it at some point, but right now it's just lying about it.